cantor-archive/src/main/java/com/salesforce/cantor/archive/s3/ArchiverOnS3.java

```diff
@@ -1,49 +1,44 @@
 package com.salesforce.cantor.archive.s3;
 
-import com.amazonaws.services.s3.AmazonS3;
 import com.salesforce.cantor.archive.file.ArchiverOnFile;
 import com.salesforce.cantor.misc.archivable.CantorArchiver;
 import com.salesforce.cantor.misc.archivable.EventsArchiver;
 import com.salesforce.cantor.misc.archivable.ObjectsArchiver;
 import com.salesforce.cantor.misc.archivable.SetsArchiver;
+import com.salesforce.cantor.s3.CantorOnS3;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.io.IOException;
 import java.util.concurrent.TimeUnit;
 
 import static com.salesforce.cantor.common.CommonPreconditions.checkArgument;
-import static com.salesforce.cantor.common.CommonPreconditions.checkString;
 
 /**
  * An implementation of the archiver which stores data on an S3 instance.
  */
 public class ArchiverOnS3 implements CantorArchiver {
     private static final Logger logger = LoggerFactory.getLogger(ArchiverOnS3.class);
-
-    protected static final String defaultBucketName = "cantor-archive";
-    protected static final long defaultChunkMillis = TimeUnit.HOURS.toMillis(1);
+    private static final String defaultArchivePathBase = "/tmp/cantor-s3-archive";
```

Comment:
let's use cwd for this; so just `cantor-s3-archive` instead of `/tmp/...`

Reply:
Yeah, I made this change in the next PR. I'll propagate it up.